Update index.html

DayserSV opened the pull request:
@SmirnovDmitrii

DayserSV committed:
Update index.html

не полная разметка

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        </ul>
+        <ul>
+          <li><button type="button">Корзина</button></li>
+          <img src="image/cart-icon.svg" alt="корзина" />
```
внутри тега ul могут быть только li.

Но я бы сделал по другому: корзина это все же не список, а отдельная ссылка и иконка относиться к ней. Можно сделать div и в него положить ссылку на корзину. Иконку потом прикрутишь через css.

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+    <main>
+      <section>
+        <h1>
+          Долго,дорого,<br />
```
<br /> тут лучше убрать потому-что он обычно им разделяют блоки а не части предложения, перенос потом можно сделать на css.

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+          Долго,дорого,<br />
+          скрупулезно.
+        </h1>
+        <h5>
```
1. заголовки идут в иерархии, как в книгах. от большего к меньшему h1 -> h2. нельзя делать 5 после 1.
2. тут это не совсем заголовок, это просто описание. подойдет <p>

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+          скрупулезно.
+        </h1>
+        <h5>
+          Математически выверенный дизайн<br />
```
`<br /> лишний`

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        <button type="button">узнать больше</button>
+      </section>
+
+      <section>
```
этот секшен лишний, картинку убери в предыдущий

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        <!---Здесь будет форма карусельки трех картинок-->
+      </section>
+
+      <section>
```
вообще эти 3 секшена можно организовать как список из 3 предложений
section
  ul
   li весь блок веб-сайты
   li весь блок приложения
   li весь блок презентации

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        </h2>
+        <p>
+          Исповедуем принципы минимализма и чистоты. Ставим математический
+          расчет<br />
```
`<br /> лишний`

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        </p>
+        <h3>выполняем заказы на разработку:</h3>
+        <ul>
+          <li>---веб-сайтов любой сложности</li>
```
--- можно убрать. потом добавишь через стили

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        <h3>с 2004 года любим точность во всем:</h3>
+        <ul>
+          <li>
+            <h2>146%</h2>
```
не соглашусь с h2, во-первых после h3 который выше нужно было бы добавить h4, а во-вторых ну не тянет на заголовок по смыслу. может p или даже лучше span, потому-что р это все же фраза больше

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,127 @@
+        <img src="#" width="210" height="88" alt="мишку" />
+      </section>
+
+      <section></section>
```
пустые теги не оставляем

DayserSV committed:
Update index.html

Сделал изменения. попытался использовать span. перекину на ноут и там дальше продолжу.

DayserSV committed:
Update index.html

вроде изменил все... попробовал спан.

DayserSV committed:
Add files via upload

папка добавлена

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,120 @@
+        <h3>с 2004 года любим точность во всем:</h3>
+        <ul>
+          <li>
+            <p><span style="font-size:150%'">146%</span></p>
```
```
тут и ниже в 3 местах 
<p><span style="font-size:150%'">146%</span></p>

заменить на

<span style="font-size:150%'">146%</span>
```

SmirnovDmitrii commented on index.html:
```diff
@@ -1,10 +1,120 @@
+        </ul>
+      </nav>
+    </header>
+    <main>
```
```
нужно сделать так
<section>
    <h1>
        Долго,дорого,<br />
        скрупулезно.
      </h1>
      <p>
        Математически выверенный дизайн для вашего сайта или мобильного
        приложения.
      </p>
      <button type="button">узнать больше</button>
        <img src="#" width="700" height="413" alt="телефоны" />
        <!---Здесь будет форма карусельки трех картинок-->
      </section>
```

DayserSV committed:
Update index.html

Поставил секцию в 30 строке. В секциях 100,104,108 убрал <p>

DayserSV committed:
Update index.html

Добавил еще одну секцию с адресом.
Добавил footer
